### PR TITLE
Add meta tags for the index page and update it according to the current challenge page

### PR DESCRIPTION
**Pull request by Sanji515**
#### Changes proposed in this pull request:

- Added default meta tags in `index.html`

- Updated these meta tags for the particular challenge with the `challenge title`, `challenge logo` and it's url and also with the `short description` of the challenge

- Link to live demo: http://pr-176-evalai.surge.sh

### CODE REVIEW

**Comment by Sanji515** [comment]
@lunayach @RishabhJain2018 @Shekharrajak @galipremsagar 

Please review

File: src/app/components/challenge/challenge.component.ts [hunk 1]
```diff
@@ -79,10 +81,10 @@ export class ChallengeComponent implements OnInit {
    * @param endpointsService  EndpointsService Injection.
    * @param challengeService  ChallengeService Injection.
    */
-  constructor(private router: Router, private route: ActivatedRoute,
+  constructor(@Inject(DOCUMENT) document: any, private router: Router, private route: ActivatedRoute,
```
Comment (lunayach) [inline src/app/components/challenge/challenge.component.ts:85]: @param DOCUMENT  Document Injection.

Comment (Sanji515) [reply src/app/components/challenge/challenge.component.ts:85]: @lunayach done

**Review by lunayach — APPROVED** [review]
Looks good to me. Functionality for the homepage could be tested with `ngrok` and it was working. But for challenges, since it required backend so I'm not sure how should we test this. But `meta` tags were there in `index.html` for the `challenges` page when seen in the inspector window, so it should work smoothly.

**Pull request merged**